### PR TITLE
OCPQE-26838 Implement 4.18 chain upgrade jobs -- part 1

### PR DESCRIPTION
**Pull request by JianLi-RH**
https://issues.redhat.com/browse/OCPQE-26838

/cc @jhou1 @jiajliu @liangxia @jianlinliu @dis

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.14.yaml [hunk 1]
```diff
@@ -88,25 +94,11 @@ tests:
...
-      - env: RELEASE_IMAGE_INTERMEDIATE415
-        name: release:intermediate415
-      - env: RELEASE_IMAGE_TARGET
-        name: release:arm64-target
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
+    - ref: cucushift-upgrade-setedge-4hops
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.14.yaml:97]: Per [comment](https://github.com/openshift/release/pull/58433#discussion_r1827248316), arm job is excluded currently for `cucushift-upgrade-setedge-4hops`

**Comment by liangxia** [comment]
The files that have changed in this PR,
```
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.1.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.2.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.3.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.4.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.5.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.8.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.9.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.10.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.14.yaml
release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.15.yaml
release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.11.yaml
release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.14.yaml
release-4.18__multi-nightly-4.18-upgrade-from-stable-4.14.yaml
release-4.18__multi-nightly-4.18-upgrade-from-stable-4.15.yaml
release-4.18__multi-stable-4.18-upgrade-from-stable-4.14.yaml 
release-4.18__multi-stable-4.18-upgrade-from-stable-4.15.yaml 
```
This seems like a partial list of chain upgrade yaml files.

**Comment by JianLi-RH** [comment]
@liangxia 
Yes, I will submit a `part 2` pr for other jobs

**Comment by liangxia** [comment]
> The files that have changed in this PR,
> 
> ```
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.1.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.2.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.3.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.4.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.5.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.8.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.9.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.10.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.14.yaml
> release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.15.yaml
> release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.11.yaml
> release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.14.yaml
> release-4.18__multi-nightly-4.18-upgrade-from-stable-4.14.yaml
> release-4.18__multi-nightly-4.18-upgrade-from-stable-4.15.yaml
> release-4.18__multi-stable-4.18-upgrade-from-stable-4.14.yaml 
> release-4.18__multi-stable-4.18-upgrade-from-stable-4.15.yaml 
> ```
> 
> This seems like a partial list of chain upgrade yaml files.

I'm fine with PR changes with part1/part2/... 
My concern is that this looks like a randomly picked file lists. It contains amd/arm/multi, and nightly/stable, but each of them are partial, which make it hard to track which part is changed while which part is not.

**Comment by JianLi-RH** [comment]
@liangxia Other files need to be changed in OCP 4.18, please check https://docs.google.com/spreadsheets/d/12NAZtZas0cHBjO3BlpdM_AuUaoYin7Na68K7SrEHvkA/edit?gid=805310624#gid=805310624

The profiles in this PR are same with OCP 4.17, in order to not block us to add chain upgrade in OCP 4.18, I submit this PR.

**Comment by JianLi-RH** [comment]
hi @liangxia please help approve this PR if no more objections

**Comment by JianLi-RH** [comment]
hi @jiajliu @liangxia , last commit updated the test frequency from f28 to f999 for 4.1, 4.2, 4.3 and 4.4 to 4.18.
BTW, when I run update-cron-entries.py on above files, it will set cron to `to be implemented`, so I did not update the cron setting.

**Comment by liangxia** [comment]
> hi @jiajliu @liangxia , last commit updated the test frequency from f28 to f999 for 4.1, 4.2, 4.3 and 4.4 to 4.18. BTW, when I run update-cron-entries.py on above files, it will set cron to `to be implemented`, so I did not update the cron setting.

I guess the script in your path might needs update. You can use soft links to make them always in sync with the repo

**Comment by JianLi-RH** [comment]
@liangxia done

**Comment by JianLi-RH** [comment]
@jhou1 Could you please add a LGTM, there is only frequency change for some config since @jiajliu gives LGTM

**Comment by jiajliu** [comment]
> hi @jiajliu @liangxia , last commit updated the test frequency from f28 to f999 for 4.1, 4.2, 4.3 and 4.4 to 4.18.

@JianLi-RH Only 4.1 has timeout issue which need to be dropped, right? From the discussion here, 4.2-4.4 paths should be kept(maybe a low frequency as a balance?) 
cc @jhou1 to confirm

**Comment by JianLi-RH** [comment]
@jiajliu FYI https://redhat-internal.slack.com/archives/CF8SMALS1/p1731898055507869?thread_ts=1731671996.685359&cid=CF8SMALS1

**Comment by jiajliu** [comment]
> @jiajliu FYI https://redhat-internal.slack.com/archives/CF8SMALS1/p1731898055507869?thread_ts=1731671996.685359&cid=CF8SMALS1

yup, I also meant the thread, based on jhou's comment, we only dropped the version without customer data?

```
For the versions that do not see customer data, we could drop them too; or alternatively set a really sparse run frequency
```

**Pull request merged**